### PR TITLE
revamp: search input in dashboard

### LINKED ISSUE #3379: revamp: search input in dashboard
**Describe the bug**
The search input in the dashboard needs to be revamped similar to the `navbar` search section.

**To Reproduce**
1. Go to https://circuitverse.org/
2. Click on the dashboard
3. Scroll down 
4. See the error

**Screenshots**
Screenshot from 2022-12-07 22-41-27

**Device Information [optional]:**

**Additional context**
Before making the PR, kindly ping your design in the slack channel!

**Are you working on this issue? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by devarsh-mavani-19**
Fixes #3379

#### Describe the changes you have made in this PR -
I have added border in scss file and added classes to search input field in index.html.erb file

### Screenshots of the changes (If any) -
Screenshot from 2022-12-18 00-07-31
Screenshot from 2022-12-17 23-19-50

### CODE REVIEW

**Comment by devarsh-mavani-19** [comment]
@vedant-jain03 Can you please check this PR?

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]
Screenshot from 2022-12-20 23-05-07


@devarsh-mavani-19 WDYT of this?

File: app/views/users/circuitverse/index.html.erb [hunk 1]
```diff
@@ -58,7 +58,7 @@
         <button id="collaborator-circuits" class="btn users-circuits-tab users-collaborators-tab"><%= t("users.circuitverse.index.collaborated_circuits") %></button>
       </div>
       <div>
-      <span><i class="fas fa-search search-icon-dashboard"></i><input type="search" class="users-circuits-search" placeholder="<%= t("users.circuitverse.index.search_circuits") %>"></span>
+      <span><i class="fas fa-search search-icon-dashboard"></i><input type="search" class="users-circuits-search form-control form-input navbar-search-bar-input" placeholder="<%= t("users.circuitverse.index.search_circuits") %>"></span>
```
[the remark this answers is not in the document]
Comment (anasaijaz) [reply app/views/users/circuitverse/index.html.erb:61]: @vedant-jain03 Can you please help me here I am confused? This branch is not merged still the issue is fixed in main branch?

[the remark this answers is not in the document]
Comment (vedant-jain03) [reply app/views/users/circuitverse/index.html.erb:61]: @anasaijaz You can update your branch and work on requested changes

**Pull request closed without merging**